lib/src/dbs/options.rs

```diff
@@ -17,15 +17,15 @@ use uuid::Uuid;
 #[derive(Clone, Debug)]
 pub struct Options {
```

Comment:
Shall we use pub(crate) rather than pub, so we can track if something becomes unused ?

Reply:
Yeah good idea @emmanuel-keller !

Reply:
Although that causes a whole load of errors, so we can do that in a future PR...